File: deaf_grandma.js
function deafGrandma() {
    let countGB = 0
    while(countGB == 0) {
        let userInput = prompt("HEY KID!","Hello Grandma!")
       
        if(userInput === "GOODBYE!"){ //case to break the first while loop
            countGB++
            alert("LEAVING SO SOON?")
            
        }else if (userInput === ""){
            alert("WHAT?")
        
        }else if(userInput === userInput.toUpperCase() && userInput !== "GOODBYE!"){
            alert("NO, NOT SINCE 1946!")
        }else {
            alert("SPEAK UP KID!")
        }

    }
    while(countGB == 1) {
        let userInput = prompt("HEY KID!","Hello Grandma!")
       
        if(userInput === "GOODBYE!"){ //case to break the 2nd while loop
            alert("LATER SKATER!")
        return
            
        }else if (userInput === ""){
            alert("WHAT?")
        
        }else if(userInput === userInput.toUpperCase() && userInput !== "GOODBYE!"){
            alert("NO, NOT SINCE 1946!")
        }else {
            alert("SPEAK UP KID!")
        }

    }
    
}
deafGrandma();

File: deaf_grandma.py
def deaf_grandma():
    count_GB = 0
    while count_GB == 0:
        # need to find a way to prompt user
        user_input = input('HEY KID!')
        if user_input == 'GOODBYE!':
            count_GB += 1
            print('LEAVING SO SOON!')
        elif user_input == '':
            print("WHAT?!")
        elif user_input == user_input.upper() and user_input != 'GOODBYE!':
            print("NO! NOT SINCE 1946!")
        else:
            print("SPEAK UP ")
    while count_GB == 1:
        user_input = input('HEY KID!')
        if user_input == 'GOODBYE!':
            print('LATER SKATER!')
            return
        elif user_input == '':
            print("WHAT?!")
        elif user_input == user_input.upper() and user_input != 'GOODBYE!':
            print("NO! NOT SINCE 1946!")
        else:
            print("SPEAK UP ")


deaf_grandma()
